server/dns_server.ml

```diff
@@ -461,13 +477,13 @@ module Notification = struct
                      Domain_name.pp zone Domain_name.pp key Ipaddr.V4.pp ip);
         cs
       end else begin
-        Log.warn (fun m -> m "replacing key zone %a oldkey %a and IP %a, new key %a"
+        Log.warn (fun m -> m "replacing key zone %a oldkey %a IP %a, new key %a"
                      Domain_name.pp zone Domain_name.pp k Ipaddr.V4.pp ip
                      Domain_name.pp key);
         cs'
       end
     | Some None ->
-      Log.info (fun m -> m "adding zone %a with key %a and IP %a (previously: no key)"
+      Log.info (fun m -> m "adding zone %a with key %a and IP %a (was no key)"
```

Comment:
Hmm, I think the old was more descriptive?

Reply:
it may have been that it did not fit on 80 characters, but i'll revise this as you wish.